magstock/__init__.py

```diff
@@ -0,0 +1,75 @@
+from uber.common import *
+from magstock._version import __version__
+
+config = parse_config(__file__)
+static_overrides(join(config['module_root'], 'static'))
+template_overrides(join(config['module_root'], 'templates'))
+
+
```

Comment:
This is how we add and override properties on the global ``c`` object.  As you can see, we're doing two things here:
* Adding a new ``c.FOOD_COUNT`` property.  This would never belong in the core Uber repo, because MAGFest doesn't sell food during the preregistration process.
* Overriding the ``c.PREREG_DONATION_OPTS`` property; as explained in the docstring, this allows us to modify the behavior of the "Want to kick in extra?" dropdown.